README.adoc

```diff
@@ -70,7 +75,7 @@ Start the `system` service by running the following command:
 mvn -pl system liberty:run
 ----
 
-Next, open another command-line session and start the `query` service by using the following command:
+Next, open another command-line session, navigate to the `finish` directory, and start the `query` service by using the following command:
```

Comment:
I suggest reminding the user to navigate to the working directory where necessary